README.md

```diff
@@ -1,26 +1,54 @@
 # Node.js GitHub Bot
 
-What's this?? ATM an experiment to see if a bot could be helpful for the Node.js project, to automate all the things!
+The Node.js Foundation's uses this bot to help manage [the repositories of the GitHub organization](https://github.com/nodejs). 
+It executes [scripts](https://github.com/nodejs-github-bot/github-bot/tree/master/scripts) in response to events that are 
+pushed to it via GitHub webhooks. All [repositories](https://github.com/nodejs) use this same bot instance and need to 
```

Comment:
"All repositories use this same bot instance" might give the impression that all repos have bot enabled. Could we be more explicit about us having to enable the bot per repo?

Something like "All repositories *who has the bot enabled*, uses the same bot instance"